gem/backends.py

```diff
@@ -148,6 +163,16 @@ def create_user(self, claims):
         username = claims.get("preferred_username")
         email = claims.get("email", "")  # Email is optional
         # We create the user based on the username and optional email fields.
+
+        # If a user already exists with this username
+        # change that user's username
+        for user in self.UserModel.objects.filter(username=username):
+            if user.profile.auth_service_uuid is None:
+                user.username = str(user.profile.site.pk) + '_' + username
+                user.save()
+            else:
+                raise FieldError
```

Comment:
Is there any way to add a message to this error? If there is then I think we should add some sort of explanation of why we're raising this error.